.travis.yml

```diff
@@ -84,7 +87,10 @@ install:
 
   # Disable autoupdate
   # * https://github.com/Homebrew/brew/blob/7d31a70373edae4d8e78d91a4cbc05324bebc3ba/Library/Homebrew/manpages/brew.1.md.erb#L202
-  - export HOMEBREW_NO_AUTO_UPDATE=1
+  #- export HOMEBREW_NO_AUTO_UPDATE=1
+  #
+  # Unset above for possible travis fix:
+  # /Users/travis/.travis/job_stages: line 166: shell_session_update: command not found
```

Comment:
As far as I understand it means that some CI stage failed. In our case I think the problem about OSX image update, need to use new toolchains.

Reply:
Ok.  You are right, it wasn't the problem (I found something that pointed to this in a web search for the error string, so I tried it blindly.).  Actually, after looking further, I think this was about the dreaded github SHA1 issue if 2017.  It couldn't find an OpenCV 3.2 release which triggered that error.  I've updated hunter in my last commit, which I hope will fix this.  If that works we can add back the `export HOMEBREW_NO_AUTO_UPDATE=1` (optimization?).

Reply:
>  If that works we can add back the export HOMEBREW_NO_AUTO_UPDATE=1 (optimization?).

Yes, this one not needed. 2-3 minutes optimization as far as I remember.

Reply:
Ok, Thanks.  That was included in the PR and release.